### PR TITLE
Issue #13: sync to pickup amp changes

### PR DESCRIPTION
**Pull request by icahoon**
Pull in Danny's amp changes. 
Update the github cloning docs.
Bump the version since this is adding the amp cli tool.

### CODE REVIEW

**Review by gstragand — APPROVED** [review]

**Review by kwelch426 — APPROVED** [review]

**Comment by icahoon** [comment]
closes issue #13

File: pkg/aws/bucket_cache.go [hunk 1]
```diff
@@ -56,11 +59,18 @@ func newBucketCache(s *storage) (*bucketCache, error) {
 	// us-east-1 is being used here because it is the default region but plays no role in
 	// listing the buckets, it is dependent on the account that created the s.s3 object.
 	region := "us-east-1"
+	log.Debug("BP1")
 	v := s.s3[region]
+	for key, val := range s.s3 {
```
Comment (danielcahoon) [inline pkg/aws/bucket_cache.go:64]: Lines 62, 64-67, 69 and 73 are leftover from debugging, they can be removed.

File: pkg/amp/account.go [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+// SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+
+package amp
+
+import (
+	"fmt"
+
+	// "github.com/cisco/arc/pkg/aaa"
```
Comment (danielcahoon) [inline pkg/amp/account.go:32]: The import of ../pkg/aaa is not used, so it can be removed.

File: pkg/aws/bucket_cache.go (hunk 1 above)
Comment (icahoon) [reply pkg/aws/bucket_cache.go:64]: I will clean these up in my next code review.

**Pull request merged**